src/Equation.jl

```diff
@@ -147,12 +147,12 @@ end
 
 
 # Get all the constants from a tree
-function getConstants(tree::Node)::Array{Float32, 1}
+function getConstants(tree::Node)::AbstractArray{ConstantType, 1}
```

Comment:
It's common to use `T` for the type in this kind of situation.

Reply:
I think `ConstantType` is meant here in the sense of a strong type / named type. I don't think `T` would be a very a meaningful name?

Reply:
Yeah I noticed later he was using T. I just saw this early and was like, is he using `ConstantType` all the way though? No he didn't, so this can be ignored.